nlp/README.md

```diff
@@ -2,40 +2,40 @@
 
 Current content:
 
--  [_Multilingual Sentence Embeddings_ (21/01/2021)](2021_01_21_multilingual_sentence_embeddings):
+-  [_Multilingual Sentence Embeddings_](multilingual_sentence_embeddings):
```

Comment:
Remove double space between the bullet point and the text (this applies to all list entries except the last).